apps/root/src/pages/position-detail/components/summary-container/components/position-data/index.tsx

```diff
@@ -279,465 +129,376 @@ const Details = ({ position, pendingTransaction, yieldOptions, totalGasSaved }:
-                    <FormattedMessage description="positionDetailsNextSwapInProgress" defaultMessage="in progress" />
-                  </Typography>
-                </DarkTooltip>
+              {DateTime.now().toSeconds() < DateTime.fromSeconds(nextSwapAvailableAt).toSeconds() ? (
+                <Typography variant="body" fontWeight={700}>
+                  {DateTime.fromSeconds(nextSwapAvailableAt).toRelative()}
+                </Typography>
+              ) : (
+                <Typography variant="body" fontWeight={700}>
+                  <FormattedMessage description="positionDetailsNextSwapInProgress" defaultMessage="in progress" />
+                </Typography>
               )}
-            </StyledDetailWrapper>
+            </ContainerBox>
           )}
-          <StyledDetailWrapper>
-            <Typography variant="body" color={baseColors.disabledText}>
-              <FormattedMessage description="positionDetailsAverageBuyPriceTitle" defaultMessage="Average buy price:" />
+        </ContainerBox>
+        <ContainerBox gap={10}>
+          <ContainerBox flexDirection="column">
+            <Typography variant="bodySmall">
+              <FormattedMessage description="initialInvestmentTotal" defaultMessage="Initial Investment Total" />
+            </Typography>
+            <ContainerBox gap={2} alignItems="center">
+              <TokenIcon size={5} token={from} />
+              <ContainerBox gap={0.5} flexWrap="wrap">
+                <Typography variant="body" fontWeight={700}>
+                  {formatCurrencyAmount(totalDeposited, from, 3)} {from.symbol}
+                </Typography>
+                <Typography variant="body">(${usdFormatter(totalDepositedPrice, 2)})</Typography>
+              </ContainerBox>
+            </ContainerBox>
+          </ContainerBox>
+          <ContainerBox flexDirection="column">
+            <Typography variant="bodySmall">
+              <FormattedMessage description="averageBuyPrice" defaultMessage="Average buy price" />
             </Typography>
-            <Typography
-              variant="body"
-              color={averageBuyPrice > 0n ? baseColors.white : baseColors.disabledText}
-              sx={{ marginLeft: '5px' }}
-            >
+            <Typography variant="body" fontWeight={700}>
               {averageBuyPrice > 0n ? (
                 <FormattedMessage
                   description="positionDetailsAverageBuyPrice"
                   defaultMessage="1 {from} = {currencySymbol}{average} {to}"
                   values={{
-                    b: (chunks: React.ReactNode) => <b>{chunks}</b>,
                     from: tokenFromAverage.symbol,
-                    to: STABLE_COINS.includes(tokenToAverage.symbol) ? 'USD' : tokenToAverage.symbol,
-                    average: formatCurrencyAmount(averageBuyPrice, tokenToAverage, 4),
+                    to: !STABLE_COINS.includes(tokenToAverage.symbol) ? tokenToAverage.symbol : '',
```

Comment:
should we have at least the `$` symbol if it is a stablecoin?

Reply:
It does! I kept the 'currencySymbol' value